packages/components/src/Badge/KaizenDraft/Badge/Badge.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { HTMLAttributes, useLayoutEffect, useState } from "react"
+import classNames from "classnames"
+import { OverrideClassName } from "@t/overrideClassName"
```

Comment:
Hmm seems the linter doesn't like this...

Reply:
Solved this, because the global `tsc` command uses a different config (without the paths in config) this doesn't work.